# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/browser/browser_103_redirect.js
# Commit: a9f3158ed688
# Full Hash: a9f3158ed6883fa156105bc4b0f78a2779c54351
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-02 21:26:33
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 9: Don't cancel redirects to insecure http early hint preloads r=necko-reviewers,valentin
#   
#   Previously those got cancelled to discourage http usage in new components.
#   Preloading them now to make this preloader behave the same way as the
#   preloader in the content process.
# ==============================================================================

diff -r 1177913e1edf -r a9f3158ed688 netwerk/test/browser/browser_103_redirect.js
--- a/netwerk/test/browser/browser_103_redirect.js	Fri Dec 02 09:45:27 2022 +0000
+++ b/netwerk/test/browser/browser_103_redirect.js	Fri Dec 02 09:45:27 2022 +0000
@@ -36,7 +36,7 @@
     "test_103_redirect_insecure_cross_origin",
     "https://example.com",
     "https://example.com/browser/netwerk/test/browser/early_hint_redirect.sjs?http://mochi.test:8888/browser/netwerk/test/browser/early_hint_pixel.sjs",
-    { hinted: 1, normal: 1 }
+    { hinted: 2, normal: 0 }
   );
 });
 
@@ -46,6 +46,6 @@
     "test_103_preload_redirect_mixed_content",
     "https://example.org",
     "https://example.org/browser/netwerk/test/browser/early_hint_redirect.sjs?http://example.org/browser/netwerk/test/browser/early_hint_pixel.sjs",
-    { hinted: 1, normal: 1 }
+    { hinted: 2, normal: 0 }
   );
 });
